idd/Energy+.idd.in

```diff
@@ -85319,42 +85334,33 @@ Table:TwoIndependentVariables,
        \note This field is used to normalize the following output data.
        \note The minimum and maximum table output fields are also normalized.
        \note If this field is blank or 1, the table data presented below will be used.
+  A7 , \field External File Name
+       \type alpha
+       \retaincase
   N8 , \field X Value #1
-       \required-field
        \type real
        \begin-extensible
   N9 , \field Y Value #1
-       \required-field
        \type real
   N10, \field Output Value #1
-       \required-field
        \type real
   N11, \field X Value #2
-       \required-field
        \type real
   N12, \field Y Value #2
-       \required-field
        \type real
   N13, \field Output Value #2
-       \required-field
        \type real
   N14, \field X Value #3
-       \required-field
        \type real
   N15, \field Y Value #3
-       \required-field
        \type real
   N16, \field Output Value #3
-       \required-field
        \type real
   N17, \field X Value #4
-       \required-field
        \type real
   N18, \field Y Value #4
-       \required-field
        \type real
   N19, \field Output Value #4
-       \required-field
```

Comment:
Now that these are not required fields, will the getinput throw an error if these fields are not present when used without an external file?